lib/Models/exercise.dart

```diff
@@ -5,7 +5,7 @@ import 'package:exerlog/UI/exercise/totals_widget.dart';
 class Exercise {
 
   Exercise(this.name, this.sets, this.bodyParts, this.totalSets, this.totalReps,
-      this.totalWeight);
+      this.totalWeight,);
```

Comment:
Don't know if we should do it in this PR or a pr after this but the entire project should probably be formatted, otherwise the trailing comma's don't really change anything.

Reply:
Preferably with flutter format so the formatting is the same for everyone and it's not changed by personal format settings.

Reply:
Yes i know, but first we should fix all linter warnings, and then think about configure some additional tools to reformat code before push to alpha (for example)